package-lock.json

```diff
@@ -2331,12 +2331,6 @@
                     "integrity": "sha1-jpQ9E1jcN1VQVOy+LtsFqhdO3hQ=",
```

Comment:
Not sure this should have been committed, I don't think any of the npm deps were changed as part of this PR.

Reply:
True dat.